hw/ip/csrng/data/csrng.hjson

```diff
@@ -504,6 +504,8 @@
       swaccess: "rw",
       hwaccess: "hro",
       hwqe: "true",
+      tags: [ // Internal HW can modify status register
+              "excl:CsrAllTests:CsrExclWrite"]
```

Comment:
sorry, I'm a bit confused. line 505 said `hwaccess: "hro"`, should we change it to another type like hrw, hwo?

Reply:
the hwaccess should be hro, since the tag is on the ERR_CODE_TEST reg, so SW will write this reg, and the internal hardware will only read from it, as in that the bits and feed them into the ERR_CODE status reg to force values on.

Reply:
Thanks for the explanation.
I think we should use `excl:CsrAllTests:CsrExclCheck` for ERR_CODE instead.
The value change of ERR_CODE may be due to `ERR_CODE_TEST` or actual error, so we have to test it in a separate test. We can't predict its value automatically in the CSR tests.

ERR_CODE_TEST is gated by `REGWEN` and we have modeled this behavior in the RAL model. If we don't exclude it, we test this behavior automatically

Reply:
Sounds good, I have made the changes to all hjson files. This also is consistent with other modules using ERR_CODE, so good suggestions.